README.md

```diff
@@ -78,8 +81,6 @@ You're all set ! Now open the `MonitorControl.xcodeproj` with Xcode! The depende
 
 `File > Packages > Resolve Package Versions`
 
-(In earlier XCode versions `Packages` menu is titled `Swift Packages`)
-
```

Comment:
This note could be removed because v4.0.0 doesn't support opening with older Xcode versions (at least, judging by the banner I got for it yesterday)

Reply:
Yes, you are right (but this is a rather artificial limit, we could downgrade the format to XCode 12 if we wanted - but I think it's better to enforce the use of the same tools if possible as otherwise things might happen that are difficult to control or understand).